Copyright (c) 2009 Ingo Schwarze <ischwarze@astaro.com>

This patch is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation; either version 2 of the License, or
(at your option) any later version.

Index: atd.c
===================================================================
--- atd.c.orig
+++ atd.c
@@ -792,7 +792,7 @@ run_loop()
     if (stat(".", &buf) == -1)
 	perr("Cannot stat " ATJOB_DIR);
 
-    if (nothing_to_do && buf.st_mtime <= last_chg)
+    if (nothing_to_do && buf.st_mtime == last_chg)
 	return next_job;
     last_chg = buf.st_mtime;
 
